package.json

```diff
@@ -2,25 +2,35 @@
   "name": "light-classnames",
   "version": "0.0.1",
   "description": "A light implementation of the classnames package with typescript support",
-  "main": "dist/index.js",
+  "author": "shahar avni",
+  "contributors": [
+    "yehuda goldberg"
+  ],
+  "license": "MIT",
+  "main": "dist/bundle.cjs.js",
+  "module": "dist/bundle.esm.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "npx jest",
```

Comment:
Didn't you want in the previous PR #1 to make a script with .d.ts and .js.map and the other one without?

Reply:
look in the file, i added `transpile` and `transpile:sourcemap` scripts.
The script `build` for publishing the package run `clean` and `transpile`.